src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -1093,6 +1094,10 @@ public void strlen(final byte[] key) {
     sendCommand(STRLEN, key);
   }
 
+  public void strAlgoLcs(StrAlgo algorithm, StrAlgoParams params) {
```

Comment:
What about `public void strAlgoLcs(StrAlgoParams params) {` and put `StrAlgo algorithm,` to `StrAlgoParams `? : )

Reply:
@dengliming thanks, agree, I will modify.